sdk/dotnet/Pulumi/Resources/InputArgs.cs

```diff
@@ -23,14 +23,22 @@ protected InputArgs()
         {
             var fieldQuery =
                 from field in this.GetType().GetFields(BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance)
-                let attr = field.GetCustomAttribute<InputAttribute>()
-                where attr != null
+                let attr1 = field.GetCustomAttribute<Pulumi.InputAttribute>()
+#pragma warning disable 618
+                let attr2 = field.GetCustomAttribute<Pulumi.Serialization.InputAttribute>()
+#pragma warning restore 618
```

Comment:
is this just for compat?  i woudl be ok with us just breaking things.  we're still in preview.